metecho/api/push.py

```diff
@@ -63,6 +64,20 @@ async def push_message_about_instance(instance, message):
         await channel_layer.group_send(group_name, sent_message)
 
 
+async def push_message_about_list(cls, message):
+    model_name = cls._meta.model_name
+    group_name = CHANNELS_GROUP_NAME.format(model=model_name, id="list")
+    channel_layer = get_channel_layer()
+
+    new_message = deepcopy(message)
+    new_message["model_name"] = model_name
+    new_message["id"] = "list"
```

Comment:
@wlonk Should `"list"` move into `constants.py`, since it's used in multiple files?